src/composers/field.tsx

```diff
@@ -20,7 +21,9 @@ export function Field<Props>(WrappedComponent: IncomingField<Props>,
     return class extends React.Component<Props & OwnProps & FormProps, { value: any }> {
 
         public state: { value: any } = {
-            value: this.props.form.getFieldValue(this.getFieldName())
+            value: (typeof this.props.form.getFieldState(this.getFieldName()) !== 'undefined')
```

Comment:
Might be better to put this in a seperate method, unreadable like this, if you do that you can extract the `getFieldState` result to a seperate variable and remove the need for a second call to the method